content/resources/known-amendments.md

```diff
@@ -105,6 +106,28 @@ The following is a list of known [amendments](amendments.html) that have been re
 
 ## Details about Known Amendments
 
+### DID
+[DID]: #did
+
+| Amendment    | DID |
+|:-------------|:----|
+| Amendment ID | TBD |
+| Status       | Open for Voting |
+| Default Vote (Latest stable release) | No |
+| Pre-amendment functionality retired? | No |
+
+Adds Decentralized Identifier (DID) functionality to the ledger that conforms to the [World Wide Web Consortium](https://www.w3.org/press-releases/2022/did-rec/) standard. DIDs provide a digital identity, not dependent on a centralized authority and controlled by the DID subject.
```

Comment:
Awkward as it sounds, "Adds to the ledger Decentralized Identifier (DID) functionality that conforms to the..." is actually clearer. As it is, it sounds like there's a misplaced modifier and it's the _ledger_ that conforms to the W3C standard.